client/extensions/hello-dolly/index.js

```diff
@@ -14,16 +14,33 @@ import Card from 'components/card';
 import SectionHeader from 'components/section-header';
 import FAQ from 'components/faq';
 import FAQItem from 'components/faq/faq-item';
+import { addReducer } from 'state';
+import reducer from './state/reducer';
+
+/**
+ * Extension init
+ */
+// TODO: Maybe make this part of the extension load/unload code for all extensions?
+addReducer( 'hello-dolly', reducer );
```

Comment:
since our naming conventions are built around `camelCase` it seems like that might be more appropriate for `helloDolly` than `kebab-case` is